### PR TITLE
Tests : Investigate intermitent error on tests (v1.1)

### LINKED ISSUE #4150: Tests : Investigate intermitent error on tests (v1.1)
image

### PR DESCRIPTION
**Pull request by David-Moreira**
Closes #4150

### CODE REVIEW

File: Tests/BasicTestApp.Client/AutocompleteMultipleReadDataComponent.razor [hunk 1]
```diff
@@ -32,15 +32,15 @@
     [Parameter]
     public List<string> SelectedTexts { get; set; }
 
-    private async Task OnHandleReadData( AutocompleteReadDataEventArgs autocompleteReadDataEventArgs )
+    private Task OnHandleReadData( AutocompleteReadDataEventArgs autocompleteReadDataEventArgs )
     {
         if ( !autocompleteReadDataEventArgs.CancellationToken.IsCancellationRequested )
         {
-            await Task.Delay( random.Next( 100 ) );
```
Comment (stsrki) [inline Tests/BasicTestApp.Client/AutocompleteMultipleReadDataComponent.razor:39]: How did we miss this??

Comment (David-Moreira) [reply Tests/BasicTestApp.Client/AutocompleteMultipleReadDataComponent.razor:39]: What do you mean?

Comment (David-Moreira) [reply Tests/BasicTestApp.Client/AutocompleteMultipleReadDataComponent.razor:39]: That's on a test component not the autocomplete itself.

Comment (stsrki) [reply Tests/BasicTestApp.Client/AutocompleteMultipleReadDataComponent.razor:39]: Ahhh. It was too early to look at the code. Ignore me :)

**Review by stsrki — APPROVED** [review]

**Pull request merged**